boards/gd32vf103c-start/include/periph_conf.h

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright (C) 2023 Gunar Schorcht <gunar@schorcht.net>
```

Comment:
Copy & paste :wink:

Reply:
Indeed. I think that the copyright still holds, as this is 99% identical to @gschorcht's code. I added myself as author, though, so that people having issues with the board send the complaints in my direction :)